build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildBinPath)\Microsoft.Common.targets" />
+  <PropertyGroup>
+    <BuildDepends>
```

Comment:
I'm open to better names. :-)

The issue is that the `Build` target *doesn't **build** anymore*. It does nothing when the cache was used, and outputs are up-to-date.

Thus, we need a target to allow devs to say "Yes, really, rebuild everything."  What should that target be called? `ReallyBuild`? For lack of imagination, I called it `_Build`, and since the *target* is `_Build`, the properties it uses are `$(_BuildDependsOn)`.